site/content/_index.md

```diff
@@ -1,5 +1,6 @@
 +++
 title = "func-e makes running Envoy® easy"
+layout = "single"
```

Comment:
this was needed right? just for my knowledge

Reply:
Yes, correct @codefromthecrypt. Without it, the content won't be assigned the correct class, hence breaking the presentation. But it's not required for the additional pages. Seems like the theme doesn't automatically assign single page layout for index page.